### DISCUSSION TITLE
Post Grid Break

### DISCUSSION THREAD

**Discussion by Jihyun22** [mmistakes/minimal-mistakes#2705] [Q&A] [answered]
The detailed explanation of the blog settings is very helpful for me to study. **Thank you!**

**but I have a problem of `post grid`setting.**
As shown in this picture, grids with 2 columns sometimes have problems of misinterpretation of posts..
![image](https://user-images.githubusercontent.com/43198705/94416698-2159a500-01ba-11eb-8cec-9240d499e00b.png)

I don't know what happened, but I think it's probably a matter of setting margins.

Below is the code of  `_sass/minimal-mistakes/_archive.scss` I modified.
Has anyone experienced the same problem as me? I'll wait for the advice of a competent person.

```
 @include breakpoint($small) {
    float: left;
    width: span(1 of 2); //5 10

    &:nth-child(2n) {
      clear: clear;
      margin-top: 0.5em;
      margin-left: 1em;
    }
  }

  @include breakpoint($medium) {
    margin-left: 1em; /* override margin*/
    margin-right: 1em; /* override margin*/
    width: span(1 of 2); //3 12

    &:nth-child(2n) {
      clear: clear;
      margin-top: 0.5em;
      margin-left: 1em;
    }
  }

```

It's like a deliberate design(!), or it's like a crazy hole...

**Comment by mmistakes** [comment, marked as answer]
I think your CSS is invalid on this line `clear: clear`

Try `clear: both` instead to properly clear the floats every 2nd item.

**Comment by Jihyun22** [comment]
Thank you! My problem has been completely solved 👍
